net/udp/Kconfig

```diff
@@ -50,11 +50,11 @@ config NET_UDP_CHECKSUMS
 		Enable/disable UDP checksum support.  UDP checksum support is
 		REQUIRED for IPv6.
 
-config NET_UDP_CONNS
-	int "Number of UDP sockets"
-	default 8
+config NET_UDP_CONNS_PER_BLOCK
+	int "Number of UDP sockets per block(one for each open)"
+	default 4
 	---help---
-		The maximum amount of open concurrent UDP sockets
+		The maximum amount of open concurrent UDP sockets per block
```

Comment:
The amount of open concurrent UDP sockets per allocation

Reply:
Done